webapp/packages/core-cli/src/dependency-validator/validateDependencies.ts

```diff
@@ -77,7 +77,7 @@ export function validateDependencies(currentPackageRoot: string) {
     }
 
     if (isTestFile) {
-      devDependencies.add('@types/jest');
+      devDependencies.add('vitest');
     }
```

Comment:
i think it will be added by default because we will use import from `vitest` so we probably can remove it